js/carousel.js

```diff
@@ -0,0 +1,21 @@
+document.addEventListener("DOMContentLoaded", function () {
+  var elem = document.querySelector(".js-flickity");
```

Comment:
By the way this is the best approach of naming the classes that are used in javascript. It should ideally start with "js-" so that when you find it html you know some javascript code is depending on it instead of searching the code every time you wonder.
Nicely done!